lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -802,20 +803,22 @@ static void on_enable_mfa(ziti_context ztx, int status, ziti_mfa_enrollment *enr
 
     struct ziti_instance_s *inst = ziti_app_ctx(ztx);
     mfa_event *ev = calloc(1, sizeof(struct mfa_event_s));
-    ev->operation = strdup(mfa_status_name(mfa_status_enrollment_challenge));
+    ev->operation = calloc(strlen(mfa_status_name(mfa_status_enrollment_challenge)), sizeof(char));
```

Comment:
This is a leak. The buffer returned from calloc is never freed and the pointer to it is overwritten on the next line.

Reply:
These mfa_events are free'd in the ziti-edge-tunnel.c class